src/lib/umd.index.ts

```diff
@@ -44,14 +44,16 @@ import UserCard from './containers/UserCard'
 import UserProfileLinks from './containers/user_profile_links/UserProfileLinks'
 import CertificationQuiz from './containers/CertificationQuiz'
 import { ProvenanceGraph } from './containers/provenance/ProvenanceGraph'
-import './style/main.scss'
 import { SynapseClient, SynapseConstants } from './utils'
 import {
   SynapseContextConsumer,
   SynapseContextProvider,
   useSynapseContext,
 } from './utils/SynapseContext'
 
+// Also include scss in the bundle
+import './style/main.scss'
+
```

Comment:
It was kinda hiding--let's make it a bit more obvious that this is also the entrypoint for our scss